src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/fixtures/UserFixture.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.fixtures
+
+import java.time.LocalDateTime
+import java.util.UUID
+import uk.gov.justice.digital.hmpps.manageusersapi.resource.external.ExternalUserDetailsDto
+
+class UserFixture {
+
+    fun createExternalUserDetails(userId: UUID, username: String, email: String): ExternalUserDetailsDto {
```

Comment:
If you make a companion object with createExternalUserDetails in it - then you can easily import the UserFixture.Companion. createExternalUserDetails in the Tests

Reply:
Done